src/visibility.js

```diff
@@ -169,6 +169,14 @@ class Visibility extends BasePlugin {
       this._playbackStartOccurred = true;
       Utils.Dom.setStyle(this._floatingPoster, 'background-image', `url("${this.player.config.sources.poster}")`);
     });
+    this.eventManager.listen(this.player, this.player.Event.ENTER_FULLSCREEN, () => {
+      let dismissibleButton = Utils.Dom.getElementBySelector('.playkit-floating-dismissible');
```

Comment:
can there be a situation where this selector returns more than one element, if there are multiple players on the page ?

Reply:
@SivanA-Kaltura 
if it's a floating play it's ok because it also have the x button.
if it's the regular player(with the floating plugin but now it in the regular mode and not the floating mode) it indeed listen to this event but it will do nothing because it doesn't have element with the class playkit-floating-dismissible.

Reply:
will the selector always return only 1 result ?